tests/conftest.py

```diff
@@ -52,3 +52,13 @@ def requests_repo_name_here(requests_mock):
     attach_requests_mock_to(requests_mock, "http://example.com/")
     attach_requests_mock_to(requests_mock, "https://repo.example.com/artifactory")
     return requests_mock
+
+
+@pytest.fixture
+def output_file(tmp_path_factory):
```

Comment:
We use these fixtures once, let's inline them to test

Reply:
Let's also use `tmp_path`, not tmp_path_factory - because the scope for the factory is session, means it doesn't remove the file after test and we need to remove the file after the test

Reply:
This way: 
```python
def test_output_table(capsys, shared_datadir, requests_mock, tmp_path):
    output_file = tmp_path / "output.txt"
```